### DISCUSSION TITLE
Mismatched Prices with Bricklink?

### DISCUSSION THREAD

**Discussion by Zag213** [rgriebl/brickstore#466] [General]
I was updating some pricing on my store today and noticed there was a price difference between the qty avg (6mo) on bricklink and the one in brickstore.  I then updated the database but still saw the price difference.  I also adjusted my settings to update the price guide if older than 1 day and still see a mismatch. 

There are several items I have noticed this on, but one of them is 90498 (black).  Bricklink shows the qty avg (6mo) at $3.77 and its $3.80 in brickstore.

Is there something im doing wrong, or is this a bug that I need to report?

Here is my system information.

### BrickStore 2021.10.2 (build: 503)

* **brickstore.ldraw**: false
* **brickstore.locale**: en
* **build.arch**: x86_64
* **build.compiler**: Microsoft Visual-C++2019 (16.1929)
* **build.date**: 2021-10-07T21:39:02.000
* **build.host**: fv-az28-314
* **build.number**: 503
* **build.qt.version**: 5.15.2
* **hw.cpu**: Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz
* **hw.cpu.arch**: x86_64
* **hw.gpu**: NVIDIA GeForce GTX 1080
* **hw.memory.gb**: 15.9
* **os.arch**: x86_64
* **os.name**: Windows 10 Version 2009
* **qt.debug**: false
* **qt.version**: 5.15.2

**Comment by rgriebl** [comment]
Please have a look at the pinned issue #80 

To get a summary for this long discussion, read this comment: https://github.com/rgriebl/brickstore/issues/80#issuecomment-839823463
